packages/tables/resources/views/components/pagination/records-per-page-selector.blade.php

```diff
@@ -1,21 +1,29 @@
 @props([
     'options',
+    'id' => 'tableRecordsPerPageSelect'
```

Comment:
Instead of passing an ID into this view, can we wrap the `<select>` in a `<label>`? Otherwise we need to generate an ID that is unique if there are multiple tables per page. So I like to avoid that